Changed gurobi from external to core

batatasfritas opened the pull request.

batatasfritas committed:
Changed gurobi from external to core

rmsds commented on configuration/packages.yaml:
```diff
@@ -95,10 +95,10 @@ packages:
+  # gurobi:
+  #   buildable: False
+  #   paths:
+  #     gurobi@8.1.1%gcc@4.8.5: /ssoft/spack/external/gurobi/8.1.1
```
I think we should actually remove these lines.

rmsds reviewed (changes requested):
Looks good, just needs the obsolete lines from `packages.yaml` to be deleted.

batatasfritas committed:
Removed the commented gurobi external lines

Following Ricardo's comment.

batatasfritas committed:
Merge branch 'releases/humagne' into packages/humagne/gurobi

nrichart committed:
Update packages.yaml

Keeping gurobi as buildable to not change the path of the already installed package

nrichart commented:
@rmsds this also should be merge for https://github.com/epfl-scitas/spack/pull/254

nrichart committed:
Update humagne.yaml

Adding gurobi@9.1.0 as core and keep 8.1.1 as external

nrichart committed:
Update packages.yaml

cleaning for gurobi

rmsds committed:
Merge branch 'releases/humagne' into packages/humagne/gurobi

rmsds committed:
setup_pr_script: add global mirror before populating temporary one

* allows us to reuse any packages already in the mirror
* particularly useful for licensed software

nrichart committed:
make failure on mirror not fatal

nrichart committed:
Update setup_pr_configuration.sh

nrichart reviewed (approved):
lgtm
